pages/community/create.tsx

```diff
@@ -0,0 +1,319 @@
+import type { NextPage } from 'next'
+import {
+  FormErrorMessage,
+  FormControl,
+  Container,
+  FormLabel,
+  Heading,
+  Select,
+  Button,
+  chakra,
+  HStack,
+  Input,
+  Image,
+  Text,
+  Flex,
+  Box,
+  useToast,
+  Spinner,
+  Link,
+} from '@chakra-ui/react'
+import Head from 'next/head'
+import {
+  CommunityCreateDto,
+  CommunityAdministratorBase,
+  CommunityClientConfigBase,
+} from 'governator-sdk';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  useFieldArray,
+  useForm,
+} from 'react-hook-form';
+import * as yup from 'yup';
+import { useSession } from 'next-auth/react';
+import { discordAxios, privateBaseAxios, privateBaseFetcher } from 'constants/axios';
+import React, { useCallback, useState, useEffect } from 'react';
+import { FiMinus, FiPlus } from 'react-icons/fi';
+
+
+const schema = yup.object({
+  name: yup.string().required('Community name is required.'),
+  administrators: yup
+    .array<{
+      user_allowlist: Array<number>;
+      provider_id: string;
+    }>()
+    .required('At least one administrator is required.'),
+  client_config: yup
+    .array<{
+      provider_id: string;
+      guild_id?: string;
+    }>()
+    .required('Client config is required.'),
+}).required();
+
+interface CommunityForm {
+  name: string;
+  administrators: Array<{
+    user_allowlist: Array<number>;
+    provider_id: string;
+  }>;
+  client_config: Array<{
+    provider_id: string;
+    guild_id?: string;
+  }>;
+}
+
+/**
+ * TODO(z):
+ * - name defaults to discord name.
+ * - admin defaults to current user.
```

Comment:
Did we confirm that admin was required?

Reply:
For creating a community? If I try to send a create community request without an admin, it will throw a 400 error. But will double check and bring reciepts.